tests/roles/development_environment/files/pre_launch.bash

```diff
@@ -86,3 +86,14 @@ fi
 if ${BASH_ALIASES[openstack]} volume show disk -f json | jq -r '.status' | grep -q available ; then
     ${BASH_ALIASES[openstack]} server add volume test disk
 fi
+
+# create another bootable volume
+if ! ${BASH_ALIASES[openstack]} volume show boot-volume ; then
+    ${BASH_ALIASES[openstack]} volume create --image cirros --size 1 boot-volume
+    wait_for_status "volume show boot-volume" "test volume 'boot-volume' creation"
+fi
+
+# Launch an instance from boot-volume (BFV)
+if ${BASH_ALIASES[openstack]} volume show boot-volume -f json | jq -r '.status' | grep -q available ; then
+    ${BASH_ALIASES[openstack]} server create --flavor m1.small --volume boot-volume --nic net-id=private bfv-server --wait
+fi
```

Comment:
this is LGTM, let's see if it works in CI.

Reply:
Looking at the CI results, seems to be working as expected

# Volume show boot-volume
+---------------------+--------------------------------------+
| Field               | Value                                |
+---------------------+--------------------------------------+
| attachments         | []                                   |
| availability_zone   | nova                                 |
| bootable            | false                                |
| consistencygroup_id | None                                 |
| created_at          | 2024-09-11T17:23:28.819553           |
| description         | None                                 |
| encrypted           | False                                |
| id                  | b23b9b40-de67-4370-927b-955ff1ce31bb |
| migration_status    | None                                 |
| multiattach         | False                                |
| name                | boot-volume                          |
| properties          |                                      |
| replication_status  | None                                 |
| size                | 1                                    |
| snapshot_id         | None                                 |
| source_volid        | None                                 |
| status              | creating                             |
| type                | tripleo                              |
| updated_at          | None                                 |
| user_id             | 746e57c749014f9a8eb85f112032b756     |
+---------------------+--------------------------------------+

Eventually the volume transitioned to available state hence we moved to server creation

openstack volume show boot-volume -f json
+ jq -r .status\n+ grep -q available
Warning: Identity file /home/zuul/install_yamls/out/edpm/ansibleee-ssh-key-id_rsa not accessible: No such file or directory.
+ ssh -q -i '~/install_yamls/out/edpm/ansibleee-ssh-key-id_rsa' root@192.168.122.100 OS_CLOUD=standalone openstack server create --flavor m1.small --volume boot-volume --nic net-id=private bfv-server --wait

# Server list shows bfv-server as ACTIVE
|+--------------------------------------+------------+--------+--------------------------------------+--------------------------+----------+
| ID                                   | Name       | Status | Networks                             | Image                    | Flavor   |
+--------------------------------------+------------+--------+--------------------------------------+--------------------------+----------+
| a0de9601-3686-48be-b16a-5a1b5afffb55 | bfv-server | ACTIVE | private=192.168.0.158                | N/A (booted from volume) | m1.small |
| 474f8b33-4383-4fcf-b2a1-5fb361a8377a | test       | ACTIVE | private=192.168.0.38, 192.168.122.20 | cirros                   | m1.small |
+--------------------------------------+------------+--------+--------------------------------------+--------------------------+----------+